addons/beehave/debug/debugger.gd

```diff
@@ -0,0 +1,116 @@
+@tool
+class_name BeehaveEditorDebugger
+extends EditorDebuggerPlugin
+
+const DebuggerTab := preload("debugger_tab.gd")
+var debugger_tab := DebuggerTab.new()
```

Comment:
Where do we free this node? I remember that anything allocated with `.new()` needs be called `.free()` manually.